### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 1 - Esercizio 17

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#476] [Esercizi M-Z]
![foglio 1 es.17](https://github.com/user-attachments/assets/5b755e7d-32b7-4e3b-bac5-9dcb7361df7a)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/c1feb218-1637-4862-923f-805d1e6608eb)
